code/components/glue/src/ConnectToNative.cpp

```diff
@@ -882,6 +917,15 @@ static InitFunction initFunction([] ()
 		{
 			UpdatePendingAuthPayload();
 		}
+		else if (!_wcsicmp(type, L"backfillDone"))

```

Comment:
Slightly risky to rely on this event to be present and handled by UI, but hopefully this shouldn't lead to any user issues.